staging/src/k8s.io/apimachinery/pkg/util/errors/errors_test.go

```diff
@@ -328,24 +328,26 @@ func TestFlatten(t *testing.T) {
 }
 
 func TestCreateAggregateFromMessageCountMap(t *testing.T) {
+	var errCount1 int32 =1
+	var errCount2 int32 =2
 	testCases := []struct {
 		name     string
 		mcm      MessageCountMap
 		expected Aggregate
 	}{
 		{
 			"input has single instance of one message",
-			MessageCountMap{"abc": 1},
+			MessageCountMap{"abc": &errCount1},
```

Comment:
this makes the test a bit harder to read. How about a helper function like `intPointer(int)*int`?